app-misc/mc/mc-4.8.33.ebuild

```diff
@@ -64,13 +64,6 @@ QA_CONFIG_IMPL_DECL_SKIP=(
 
 src_prepare() {
 	default
-
-	# Bug #906194, #922483
-	if use elibc_musl; then
-		eapply "${FILESDIR}"/${PN}-4.8.30-musl-tests.patch
-		eapply "${FILESDIR}"/${PN}-4.8.31-musl-tests.patch
-	fi
-
```

Comment:
Please remove the whole `src_prepare` function, it is useless without these 2 patches (no need to `eautoreconf` anymore).

I tested on musl and the tests do indeed pass without the patches.

Reply:
Sure, I only didn't do that because some distros insist on generating the build system with their own version of autotools. But otherwise I am happy to do it.